scripts/docbuild.rb

```diff
@@ -30,5 +30,6 @@ def execute(cmd)
 
 $log.info("Copying docs to fileshare...")
 FileUtils.rm_rf DOCS_FILESHARE
+FileUtils.mkdir_p DOCS_FILESHARE
```

Comment:
I am intentionally copying the DOCS_PACKAGE to DOCS_FILESHARE to remove the level of indirection, though maybe that's me being overly cautious...I think this is actually a bug in asciibinder when it packages.